packages/RatPack.package/RPSessionMiddleware.class/instance/tokenPayload..st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+tokenPayload: token
+
+	| separatedToken |
+	separatedToken := token subStrings: '.'.
+	
+	^ (RPJsonContentParser parse: separatedToken second base64Decoded)
```

Comment:
Here, we would need to do base64UrlDecoding. Not undoing the changes we did in the encoding process could break this.